package.json

```diff
@@ -16,21 +16,22 @@
   "author": "Aller Media",
   "license": "ISC",
   "peerDependencies": {
-    "babel-cli": "6.26.0",
-    "babel-preset-env": "1.6.1",
-    "babel-preset-react": "6.24.1",
+    "@babel/cli": "7.0.0-beta.44",
+    "@babel/core": "7.0.0-beta.44",
     "styled-components": "^3.2.5"
   },
   "dependencies": {
     "@aller/eslint-config-aller": "0.5.2",
+    "@babel/plugin-transform-object-assign": "7.0.0-beta.44",
+    "@babel/preset-env": "7.0.0-beta.44",
+    "@babel/preset-react": "7.0.0-beta.44",
+    "@babel/preset-stage-0": "7.0.0-beta.44",
     "@maji/react-prism": "1.0.1",
     "@storybook/addon-actions": "3.4.4",
     "@storybook/addon-links": "3.4.4",
     "@storybook/react": "3.4.4",
     "aurora-deep-slice-merge": "github:soldotno/aurora-deep-slice-merge",
     "babel-plugin-styled-components": "1.5.1",
-    "babel-plugin-system-import-transformer": "3.1.0",
```

Comment:
is this removed  intentionally?

Reply:
This package does not work with babel 7 it seems, so had to remove it. Hence the use of `require()` instead of `import()` :(